package.json

```diff
@@ -89,7 +89,7 @@
     "webpack-cli": "3.1.1",
     "webpack-dev-server": "3.1.9",
     "whatwg-fetch": "^2.0.2",
-    "wp-calypso": "github:automattic/wp-calypso#ef6cd7465e7c5dce08ce5bacb65677e19c7ca309",
```

Comment:
The commit is already on `master`, let's use the `master` refspec.

Reply:
Using 👍